README.md

```diff
@@ -98,20 +98,108 @@ const styles = StyleSheet.create({
 
 Your app is now ready to launch. Run the app in Xcode or type `react-native run-ios` in the terminal.
 
+### Usage
+
+There are 2 different ways on how to use the PSPDFKit React Native wrapper on iOS. 
+- Present a document via a Native Module modally.
+- Show a PSPDFKit view via a Native UI component.
+
+Depending on your needs you might want to use one or the other.
+
+### Native Module
+
+Using the Native Module (`PSPDFKit.present()`), you can present a document with PSPDFKit modally in fullscreen.
+You can specify the path to the document you want to present, and [configuration options](#configuration).
+
+```javascript
+...
```

Comment:
I would keep the imports for easy copy/paste. In other words, it should be easy to replace the whole content of `App.js` with the content of the code block.